### DISCUSSION TITLE
Notifications not working

### DISCUSSION THREAD

**Discussion by mvienna** [bkaankose/Wino-Mail#332] [Q&A] [answered]
Hello, I'm experiencing an issue with Wino Mail. I'm not receiving notifications for new emails; they (email's) only appear when I manually open the Wino Mail client. Is this the expected behavior, or is there a setting I need to adjust to enable real-time notifications? 

Windows 11 / IMAP SSL
1.7.8.0

**Comment by bkaankose** [comment, marked as answer]
This issue should be fixed with the latest 1.9.0+ releases. We changed the whole background synchronization system that feeds the notifications.
